### PR TITLE
Optional log details

### PR DESCRIPTION
**Pull request by PeachesMLG**
Made logging details optional

### CODE REVIEW

File: src/main/java/com/j256/ormlite/table/TableUtils.java [hunk 1]
```diff
@@ -346,26 +355,30 @@ public static int createTable(Dao<?, ?> dao) throws SQLException {
...
 		} else {
 			TableInfo<T, ID> tableInfo = new TableInfo<T, ID>(databaseType, dao.getDataClass());
-			return doCreateTable(connectionSource, tableInfo, ifNotExists);
+			return doCreateTable(connectionSource, tableInfo, ifNotExists, logDetails);
 		}
 	}
 
 	private static <T, ID> int doCreateTable(ConnectionSource connectionSource, TableInfo<T, ID> tableInfo,
-			boolean ifNotExists) throws SQLException {
+											 boolean ifNotExists, boolean logDetails) throws SQLException {
```
Comment (noordawod) [inline src/main/java/com/j256/ormlite/table/TableUtils.java:373]: Your IDE is set to insert tabs, please fix this line.

File: src/main/java/com/j256/ormlite/table/TableUtils.java [hunk 2]
```diff
@@ -375,6 +388,11 @@ public static int createTable(Dao<?, ?> dao) throws SQLException {
 
 	private static int doStatements(DatabaseConnection connection, String label, Collection<String> statements,
 			boolean ignoreErrors, boolean returnsNegative, boolean expectingZero) throws SQLException {
+	    return doStatements(connection, label, statements, ignoreErrors, returnsNegative, expectingZero, true);
+	}
+
+	private static int doStatements(DatabaseConnection connection, String label, Collection<String> statements,
+									boolean ignoreErrors, boolean returnsNegative, boolean expectingZero, boolean logDetails) throws SQLException {
```
Comment (noordawod) [inline src/main/java/com/j256/ormlite/table/TableUtils.java:395]: Too many tabs here.

File: src/main/java/com/j256/ormlite/table/TableUtils.java (hunk 1 above)
Comment (noordawod) [inline src/main/java/com/j256/ormlite/table/TableUtils.java:373]: Too many tabs here.

File: src/main/java/com/j256/ormlite/table/TableUtils.java [hunk 3]
```diff
@@ -383,9 +401,11 @@ private static int doStatements(DatabaseConnection connection, String label, Col
 				compiledStmt = connection.compileStatement(statement, StatementType.EXECUTE, noFieldTypes,
 						DatabaseConnection.DEFAULT_RESULT_FLAGS, false);
 				rowC = compiledStmt.runExecute();
-				logger.info("executed {} table statement changed {} rows: {}", label, rowC, statement);
+				if(logDetails) {
```
Comment (noordawod) [inline src/main/java/com/j256/ormlite/table/TableUtils.java:404]: Insert a space before the opening brace.

**Review by noordawod — CHANGES_REQUESTED** [review]
Thanks for the pr. All in all it looks okay, few cosmetic changes required though.

**Comment by j256** [comment]
I'm curious why this is necessary?  Is this more about too much log output or what?

**Comment by PeachesMLG** [comment]
> I'm curious why this is necessary? Is this more about too much log output or what?

Ye basically, Im coding a java program that uses this and people have asked if its possible that it doesnt show the creation commands since i have like 15 tables and it spams the console a bit

**Comment by PeachesMLG** [comment]
Although I just realised in your latest commit you added setLogBackendType so I can just do setLogBackendType(LogBackendType.NULL)

Unfortunatly this hasn been released to maven central yet, when do you plan on posting that new version?

**Comment by PeachesMLG** [comment]
Guess ill hoest the repo myself until you release it :) Thanks

**Pull request closed without merging**